src/DataFixtures/ActivityFixtures.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace App\DataFixtures;
+
+use App\Entity\Activity;
+use App\Entity\Technology;
+use App\Entity\Type;
+use App\Entity\User;
+use DateTime;
+use Doctrine\Bundle\FixturesBundle\Fixture;
+use Doctrine\Common\DataFixtures\DependentFixtureInterface;
+use Doctrine\Common\Persistence\ObjectManager;
+use Faker\Factory;
+
+class ActivityFixtures extends Fixture implements DependentFixtureInterface
+{
+    public function load(ObjectManager $manager): void
+    {
+        $faker = Factory::create();
+        for ($i = 0; $i < 10; $i++) {
+            $activity = new Activity();
+            $activity->setName($faker->jobTitle);
+            $activity->setDescription($faker->sentence);
+            $activity->setApplicationDeadline($faker->dateTimeInInterval('now', '+10 days'));
+            $activity->setFinalDeadline($faker->dateTimeInInterval('+10 days', '+30 days'));
+            $activity->setStatus(array_rand($activity->getAllStatuses()));
```

Comment:
`array_rand` returns a random index not a random value. Even if it works in your case it's wrong. You could do `$activity->getAllStatuses(array_rand($activity->getAllStatuses()))` or `$faker->randomElement($activity->getAllStatuses())`. After changing the function getAllStatuses to static you will need to use `Activity::getAllStatuses()` instead of `$activity->getAllStatuses()`.